Automattic/gravatar
# Hovercards: Format files

## Proposed Changes

This PR reformats the hovercard files to comply with the latest linter settings.

With that, I believe https://github.com/Automattic/gravatar/issues/20 can be closed.

## Testing Instructions

* Checkout this branch
* Make sure the node_modules are up to date running `npm install`
* Still in the terminal, navigate to the hovercards folder: `cd web/packages/hovercards`
* Run `npm run lint`
* There shouldn't be any lint errors
